pep_sphinx_extensions/pep_zero_generator/writer.py

```diff
@@ -177,24 +180,25 @@ def write_pep0(self, peps: list[PEP], header: str = HEADER, intro: str = INTRO,
         # PEP types key
         self.emit_title("PEP Types Key")
         for type_ in sorted(TYPE_VALUES):
-            self.emit_text(f"    {type_[0]} - {type_} PEP")
+            self.emit_text(
+                f"* **{type_[0]}** --- *{type_}*: {ABBREVIATED_TYPES[type_]}"
+            )
             self.emit_newline()
 
+        self.emit_text(":pep:`More info in PEP 1 <1#pep-types>`.")
         self.emit_newline()
 
         # PEP status key
         self.emit_title("PEP Status Key")
         for status in sorted(STATUS_VALUES):
             # Draft PEPs have no status displayed, Active shares a key with Accepted
-            if status in HIDE_STATUS:
-                continue
-            if status == STATUS_ACCEPTED:
-                msg = "    A - Accepted (Standards Track only) or Active proposal"
-            else:
-                msg = f"    {status[0]} - {status} proposal"
-            self.emit_text(msg)
+            status_code = "<none>" if status == STATUS_DRAFT else status[0]
```

Comment:
We might want to bikeshed a bit over this. E.g. we could use the space _symbol_, which as a single character like the others would make the bullets line up (which the previous throws off)

```suggestion
            status_code = "␣" if status == STATUS_DRAFT else status[0]
```

Reply:
Can't think of a better one, but "<none>" seems better than "␣" 🤷

Reply:
I guess so, from a clarity perspective, though it still irks me that it throws the output formatting off...I was originally going to propose using another letter/symbol for Draft (and maybe `Ac`/`At` for Accepted/Active) since we're special casing it anyway, but I didn't want to drag this PR further out of scope than I have already.

Also, fun fact: apparently [`<none>`] displays as [<none>] in GFM. TIL...

Reply:
An alternate, more verbose but the alignment is already lost.
```suggestion
            status_code = "No status key" if status == STATUS_DRAFT else status[0]
```

Reply:
The "key" here is this _section_, listing the mappings from letters to statuses/types, so saying there is no key in the key is a self-defeating prophecy :) Instead, you could say "No letter" or "No status letter". Also, the `<>` makes clear that it is a description rather than a literal, as the other entries are. So:

```suggestion
            status_code = "<No letter>" if status == STATUS_DRAFT else status[0]
```

or

```suggestion
            status_code = "<No status letter>" if status == STATUS_DRAFT else status[0]
```